common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,220 @@
+
+#culture_group_regional
+####################
+##Tribal##
+####################
+innovation_amphibious = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		OR = {
+			##Nagas
+			this = culture:naga
+			##Murlocs
+			this = culture:murloc
+			this = culture:gorloc
+			##Giblin
+			this = culture:giblin
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -1
+		supply_duration = 0.2
+		no_water_crossing_penalty = yes
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		OR = {
+			##Human
+			this = culture:tirassian
+			##Vrykul
+			this = culture:vrykul
+			this = culture:kvaldir
+			this = culture:hyldnir
+			this = culture:frost_vrykul
+			this = culture:drust
+			##Pirates
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.4
+		raid_speed = 0.2
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_inner_peace = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		 OR = {
+			##Pandaren
+			this = culture:mainland_pandaren
+			this = culture:wandering_pandaren
+		  }
+	}
+	modifier = {
+		monthly_lifestyle_xp_gain_mult = 0.20
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_hive_breeding = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @misc_inventions
+	potential = {
+		 OR = {
+			##Demonic
+			this = culture:qiraji
+			this = culture:mantid
+			this = culture:nerubian
+
+	
+		}
+	}
+	modifier = {
+		levy_reinforcement_rate = 0.3
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+
+####################
+##Early Medieval##
+####################
+innovation_time_is_money = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @misc_inventions
+	potential = {
+		##Goblin
+		this = culture:goblin	
+	}
+	modifier = {
+		monthly_income = 5
+		vassal_tax_contribution_mult = 0.3
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regular
+}
+innovation_elite_ranks = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @misc_inventions
+	potential = {
+		  OR = {
+			##Human
+			this = culture:azerothian
+			this = culture:gilnean
+			this = culture:arathorian
+			this = culture:hillsbrad
+			this = culture:lordaeronian
+			this = culture:dalaranian
+			this = culture:alteraci
+			this = culture:baradin
+			this = culture:northern
+			this = culture:theramore
+			this = culture:western
+	
+		}
+	}
+	modifier = {
+		knight_limit = 10
```

Comment:
This is A HUGE BONUS. You must balance all these innovations as we did it with holy sites and buildings on this spreadsheet.
https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit?usp=sharing